test/test_proxy_tensor.py

```diff
@@ -732,6 +732,17 @@ def f(x, y):
         x, y = torch.randn(2), torch.randn(2)
         self.assertEqual(g(x, y), f(x, y))
 
+    def test_alias(self):
+        def f(x):
+            return torch.ops.aten.alias(x)
+
+        r = str(make_fx(f, tracing_mode="fake")(torch.randn(2)).code).strip()
+        # NB: this should not have a detach call
+        self.assertExpectedInline(r, """\
+def forward(self, x_1):
+    alias = torch.ops.aten.alias.default(x_1);  x_1 = None
+    return alias""")
+
```

Comment:
It doesn't seem that alias would usually go the path with detach anyway. What happens differently when we trace with proxy tensor?

Reply:
Prior to this PR, there was a detach call after the alias.

Reply:
Ostensibly, this is because the tensor was not unique() for some reason, but I didn't investigate why.

Reply:
Is the goal to avoid seeing these detaches in the trace? If the caller does the dispatch now, naively I would expect the detach to still be there.

Reply:
Yes. I agree that if a caller detaches, we should see it. But ADInplaceOrView is a very "internal" function and I don't think it should show.

Reply:
Hmm I see thanks, I might still be confused about what you mean by caller though (see my other comment)